website/static/files/build_orca_reconcile.sql

```diff
@@ -6,150 +6,141 @@ SET search_path TO orca, public;
+  COMMENT ON COLUMN reconcile_s3_objects.key_path IS 'Full path and file name of the object in the S3 bucket from the S3 list command.';
+  COMMENT ON COLUMN reconcile_s3_objects.etag IS 'AWS etag value from the S3 list command.';
+  COMMENT ON COLUMN reconcile_s3_objects.last_update IS 'AWS Last Update from the S3 list command.';
+  COMMENT ON COLUMN reconcile_s3_objects.size_in_bytes IS 'AWS size of the file in bytes from the S3 list command.';
+  COMMENT ON COLUMN reconcile_s3_objects.storage_class IS 'AWS storage class the object is in from the S3 list command.';
+  COMMENT ON COLUMN reconcile_s3_objects.delete_marker IS 'Set to `True` if object is a delete marker.';
 
-    -- Create reconcile_orphan_report table
-    CREATE TABLE IF NOT EXISTS reconcile_orphan_report
-    (
-      job_id                  int8 NOT NULL
-    , orca_archive_location   text NOT NULL
-    , key_path                text NOT NULL
-    , etag                    text NOT NULL
-    , last_update             timestamp with time zone NOT NULL
-    , size_in_bytes           int8 NOT NULL
-    , storage_class           text NOT NULL
-    , CONSTRAINT PK_reconcile_orphan_report_1 PRIMARY KEY(job_id,orca_archive_location,key_path)
-    , CONSTRAINT reconcile_jobs_orphan_report_fk FOREIGN KEY(job_id) REFERENCES reconcile_jobs(id)
-    );
 
-    COMMENT ON TABLE reconcile_orphan_report
-      IS 'Table that identifies objects in the ORCA S3 Glacier bucket that are not in the ORCA catalog from the internal reconciliation job.';
-    COMMENT ON COLUMN reconcile_orphan_report.job_id
-      IS 'Associates the orphaned file to a internal reconcilation job. Refrences the reconcile jobs table.';
-    COMMENT ON COLUMN reconcile_orphan_report.orca_archive_location
-      IS 'ORCA S3 Glacier bucket the orphan was found in. Value is obtained from the reconcile_s3_objects (orca_archive_location) column.';
-    COMMENT ON COLUMN reconcile_orphan_report.key_path
-      IS 'Key that contains the path and file name. Value is obtained from the reconcile_s3_objects (key_path) column.';
-    COMMENT ON COLUMN reconcile_orphan_report.etag
-      IS 'AWS Etag of the object. Value is obtained from the reconcile_s3_objects (etag) column.';
-    COMMENT ON COLUMN reconcile_orphan_report.last_update
-      IS 'AWS last update of the object. Value is obtained from the reconcile_s3_objects (lst_update) column.';
-    COMMENT ON COLUMN reconcile_orphan_report.size_in_bytes
-      IS 'AWS size of the object in bytes. Value is obtained from the reconcile_s3_objects (size_in_bytes) column.';
-    COMMENT ON COLUMN reconcile_orphan_report.storage_class
-      IS 'ASWS storage class the object is in. Value is obtained from the reconcile_s3_objects (storage_class) column.';
+  CREATE TABLE reconcile_catalog_mismatch_report  ( 
+    job_id                  	int8 NOT NULL,
+    collection_id           	text NOT NULL,
+    granule_id              	text NOT NULL,
+    filename                	text NOT NULL,
```

Comment:
Wondering if we should drop filename since key_path already has that information in it. Maybe shorten key_path to key? Thoughts?

Reply:
Our standard in several other tables is "key_path". I seem to recall during recovery architecture we discussed that having both key_path and filename kept us flexible for other storage methods.